### ISSUE TITLE
value offset between PC7/Garmin and GC training mode

### ISSUE THREAD

**Issue by Rennschnecke21805** [GoldenCheetah/GoldenCheetah#1905]
When using GC in training mode and PC7 in parallel power values are constantyl apporx. 40 W higher in GC training mode than they are on the PC7. Both devices receive the same ANT+ signal from an SRM Powermeter and all other values received via ANT+ (heart rate, cadence, speed) are consistent between the two devices...
This leaves me kind of puzzled. Is GC applying any kind of algorithm on the power data ?
Actually I previously had the notion that GC is interpreting power datat differently depending on whether they are imported from a file od downloaded directly from the device in GC...

**Comment by liversedge** [comment]
I assume you are using SRM cranks?

The slope + zero offset are not being picked up by train mode -- its a serious issue and we are looking to finally resolve it in v4.

I'm closing this only because it would be a duplicate of the bug report to address these issues.

Thanks,
Mark

**Comment by Rennschnecke21805** [comment]
Well yes - and sorry for not going through the bug reports... ;-)

**Comment by liversedge** [comment]
No probs, this is one of those really rubbish bugs. The hardcoded values are from my cranks.
So it doesn't affect me..................
